apps/patient/src/components/BrandedPharmacyCard.tsx

```diff
@@ -18,14 +17,13 @@ import capsulePharmacyIdLookup from '../data/capsulePharmacyIds.json';
 
 interface Props {
   pharmacyId: string;
-  selectedId: string;
-  handleSelect: (id: string) => void;
+  selectedId?: string;
+  handleSelect?: (id: string) => void;
 }
 
 const PHARMACY_BRANDING = {
   [process.env.REACT_APP_AMAZON_PHARMACY_ID as string]: {
-    logo: amazonPharmacyLogo,
-    description: 'Save time. Save money. Stay healthy.'
```

Comment:
Why'd we drop this?

Reply:
just to distinguish a bit more from same day, [chat w/ jomi](https://photonhealth.slack.com/archives/C0302SQME1K/p1713388912122989?thread_ts=1713387121.069459&cid=C0302SQME1K)

Reply:
reverted to original